conf/modules/genome_assembly.config

```diff
@@ -24,6 +24,15 @@ process {
         ]
     }
 
+    withName: '.*:ASSEMBLY:HIFIASM' {
+
+        ext.args = { "${params.extra_hifiasm_options}" }
+
+        publishDir = [
+            enabled: false,
+        ]
+    }
```

Comment:
Just being super picky but it would be nice if the change of line in the configs was kept  the same (see conf/modules/call_paralogs.config), not necessary to take in this PR if you don't want too though 😄

Reply:
I agree, not that many changes left so let's just do them in this PR.